contracts/pcv/PCVGuardian.sol

```diff
@@ -86,6 +92,10 @@ contract PCVGuardian is IPCVGuardian, CoreRef {
             pcvDeposit._pause();
         }
 
+        if (depositAfter) {
+            IPCVDeposit(safeAddress).deposit();
```

Comment:
I think we might want to use a try catch here. If we're withdrawing a deposit to a safe address it's likely to be urgent/an emergency, and we don't want to risk the deposit part of this withdrawal reverting the entire thing.

Reply:
Disagree, I think it try catch is an extremely dangerous pattern and this is code that should fail loud.

Reply:
Plus in an emergency, Defender would alert us if this would fail in which case we just do false.

Reply:
The scenario in which we're using this is both Defender and manual. Either way, if we're using it, it means that *something has gone wrong* and we need to pull funds out of the danger zone, immediately. Depositing into the underlying deposit is sugar on top and isn't necessary and is another possible point of failure.

Reply:
Try catch isn't inherently dangerous. It's only dangerous if it's a problem if only part of your code succeeds. Here, it's not.

Reply:
> Depositing into the underlying deposit is sugar on top and isn't necessary and is another possible point of failure. 

This is the part where we are sticking. Its not sugar on top, its critical functionality to have atomically. Thats why I made this PR. In many cases undeposited tokens aren't counted in CR oracle. In a perfect storm, this could open up the backstop or change risk curve dramatically. Better to control these variables.